openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -249,7 +249,7 @@ Credential Format Profiles are defined as follows:
 This profile defines the following additional requirements for IETF SD-JWT VCs as defined in [@!I-D.ietf-oauth-sd-jwt-vc].
 
 * Compact serialization MUST be supported as defined in [@!I-D.ietf-oauth-selective-disclosure-jwt]. JSON serialization MAY be supported.
-* It is at the discretion of the Issuer whether to use `exp` claim and/or a `status` claim to express the validity period of an SD-JWT VC. The Wallet and the Verifier MUST support both mechanisms.
+* It is RECOMMENDED that Issuers limit the validity period when issuing SD-JWT VC. If the Issuer wishes to do so, it MUST use an `exp` claim, a `status` claim, or both. The Wallet and the Verifier MUST support both mechanisms.
```

Comment:
This might be playing semantics, but I find that "wish" further weakens the recommendation to do so. Maybe keep it cleaner as follows:

```suggestion
* It is RECOMMENDED that Issuers limit the validity period when issuing SD-JWT VC. When doing so, the Issuer MUST use an `exp` claim, a `status` claim, or both. The Wallet and the Verifier MUST support both mechanisms.
```